corehq/apps/locations/tests/test_dbaccessors.py

```diff
@@ -45,10 +58,20 @@ def make_user(name, location):
         )
         cls.george.set_location(cls.domain, cls.meereen)
 
+        for user in [cls.varys, cls.tyrion, cls.daenerys, cls.george]:
+            cls._send_user_to_es(user)
+
+    @classmethod
+    def _send_user_to_es(cls, user):
+        with patch('corehq.pillows.user.get_group_id_name_map_by_user', return_value=[]):
+            send_to_elasticsearch('users', transform_user_for_elasticsearch(user.to_json()))
```

Comment:
Doesn't this already happen synchronously on save for users? 
https://github.com/dimagi/commcare-hq/blob/534d8b43ee78b6ab1959047737f86160afc97fc5/corehq/apps/users/signals.py#L40-L49

Reply:
Updated.